contributing/code/bc.rst

```diff
@@ -258,6 +258,14 @@ Make public or protected                                                  Yes
 Remove private property                                                   Yes
 **Constructors**
 Add constructor without mandatory arguments                               Yes             :ref:`[1] <note-1>`
+:ref:`Add argument without a default value <add-argument-public-method>`  No
+Add argument with a default value                                         Yes
+Remove argument                                                           No              :ref:`[3] <note-3>`
+Add default value to an argument                                          Yes
+Remove default value of an argument                                       No
+Add type hint to an argument                                              No
+Remove type hint of an argument                                           No
+Change argument type                                                      No
```

Comment:
These 3 are allowed, if the new type is compatible with the previous type (e.g. you can change from `SomeObject` to `SomeObject|OrAnotherOne`, but not to `string`).

Reply:
why would public final methods use a plain `Yes` ? The restriction is not about inheritance here, so it is not different for final constructors.